HandsOnTutorial/1-hello-world.fsx

```diff
@@ -67,26 +67,4 @@ In case you run into trouble, this can be used to clear all process
 records in the cluster: 
 *)
 
-cluster.ClearAllProcesses()
-
-(**
-If you need to get really heavy, you can reset the cluster, which clears 
-all process state in queues and storage. Other storage is left unchanged.
-Your worker roles may need to be manually rebooted (e.g. from the Azure 
-management console).
-*)
-// cluster.Reset()
-
-(** You can add your local machine to be a worker in the cluster. *)
-
-// MBraceAzure.SpawnLocal(config, 4)
-
-(** You can optionally look at worker (not client) logs for the last 5 minutes. *)
-
-// cluster.ShowSystemLogs(300.0)
-
-(** In this tutorial, you've run your very first work on an MBrace cluster
```

Comment:
Please reinstate all the terminating comments that wrap up the tutorial.  These are the "summary" sections at the end of each page, e.g. http://www.m-brace.net/azure/HandsOnTutorial/1-hello-world.html.  They can be improved, but a concluding summary wrapping up what has been learnt is needed, so please make sure they are there.